sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -221,8 +221,8 @@ class HiveDDLSuite
         sql(
           s"""
              |ALTER TABLE $tab ADD
-             |PARTITION (ds='2008-04-08', hr=11) LOCATION '$part1Path'
-             |PARTITION (ds='2008-04-08', hr=12) LOCATION '$part2Path'
+             |PARTITION (ds='2008-04-08', hr=11) LOCATION '${part1Path.toURI}'
```

Comment:
It seems due to the parser. If the path is something like `C:\tmp\b\c`, then, it goes like `C:    mpbc` (escaping). To deal with this, we should make it `C:\\tmp\\b\\c` or URI. The simplest choice seems to use URI unless it is dedicated test to such case.